src/test/java/api/loans/scenarios/CheckInDeclaredLostItemTest.java

```diff
@@ -19,13 +19,26 @@
 import org.folio.circulation.support.http.client.Response;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 import api.support.builders.CheckInByBarcodeRequestBuilder;
 import io.vertx.core.json.JsonObject;
 
 public class CheckInDeclaredLostItemTest extends RefundLostItemFeesTestBase {
 
+  @Before
+  public void setupLoanType() {
+    noteTypeFixture.generalNoteType();
+  }
```

Comment:
The name refers to a loan type yet the implementation refers to a note type, could this be renamed to remove this discrepancy?